modules/core/src/iterators/make-stream/make-dom-stream.ts

```diff
@@ -34,6 +34,7 @@ export function makeStream<ArrayBuffer>(
             controller.close();
           } else {
             // TODO - ignores controller.desiredSize
+            // @ts-ignore
```

Comment:
Should change this into a `@ts-expect-error`. I got an error about this originally but maybe it's now fixed on master.